fe/fe-common/src/main/java/org/apache/doris/catalog/Type.java

```diff
@@ -73,6 +73,7 @@ public abstract class Type {
     public static final ScalarType DATEV2 = new ScalarType(PrimitiveType.DATEV2);
     public static final ScalarType TIMEV2 = new ScalarType(PrimitiveType.TIMEV2);
     public static final ScalarType STRING = ScalarType.createStringType();
+    public static final ScalarType VARBINARY = ScalarType.createVarbinaryType(-1);
```

Comment:
Please add this new type to this Set: 
org.apache.doris.statistics.ColumnStatistic.UNSUPPORTED_TYPE